recipes/devel/host-compat-toolchain.yaml

```diff
@@ -6,7 +6,7 @@
 shared: True
 
 environment:
-    AUTOCONF_TARGET: "$(gen-autoconf,bob_compat)"
+    AUTOCONF_TARGET: "${SANDBOX_AUTOCONF_TARGET:-$(gen-autoconf,bob_compat)}"
```

Comment:
this is wrong! host-compat doesn't match, if we have a cross-compiled sandbox!
that was not a requirement so far. maybe this would be work:

`${AUTOCONF_TARGET:-$(gen-autoconf,bob_compat)}`

use host, if not already set...?!